packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -356,26 +337,32 @@ contract Deploy is Deployer {
         // enabled to prevent a nastier refactor to the deploy scripts. In the future, the L2OutputOracle will be
         // removed. If fault proofs are not enabled, the DisputeGameFactory proxy will be unused.
         deployERC1967Proxy("DisputeGameFactoryProxy");
-        deployERC1967Proxy("L2OutputOracleProxy");
```

Comment:
moving this out into the main `_run()` function, so that `deployOpChain()` is closer to `DeployOPChain.run()`